image_analysis/service.py

```
@@ -0,0 +1,146 @@
+            print(f"[Worker ERROR] job_id={job_id}, error={exc}")
+        finally:
+            _active_users.discard(user_id)
+            _job_store.pop(job_id, None)
+            _job_queue.task_done()
+            _cleanup_old_jobs()
+
+
+async def start_queue_workers() -> None:
+    get_pool()
+    for _ in range(_WORKER_CONCURRENCY):
+        asyncio.create_task(_queue_worker())
```

Ruff (0.15.11)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)